typescript/sdk/src/token/EvmERC20WarpRouteReader.ts

```diff
@@ -156,6 +157,7 @@ export class EvmERC20WarpRouteReader extends EvmRouterReader {
           this.provider,
         ).allowedRebalancers();
 
+        // Set to undefined if the object is empty because might trigger false positives in the checker
```

Comment:
this makes sense but it might be more clear if we treat undefined and empty as the same in the diff checker?

Reply:
rather than pruning from the derived config